### DISCUSSION TITLE
Akita 8 Angular devtools

### DISCUSSION THREAD

**Discussion by jongbonga** [salesforce/akita#1058] [Q&A] [answered]
I think the previous implementation of `AkitaNgDevtools` was pretty simple, one-liner `environment.production ? [] : AkitaNgDevtools.forRoot()` in the app module import.

Since I have updated to Akita 8, I can't figure out how to use the [new implementation for Angular](https://opensource.salesforce.com/akita/docs/enhancers/devtools#usage). where do you place the provided code snippet?

Can someone guide me to the right direction please?

**Comment by Den-dp** [comment, marked as answer]
Call `provideAkitaDevtools()` inside your providers array like that: https://github.com/salesforce/akita/blob/dc91896d6930551b5c14abd5dfa16fada1425f8e/packages/ng-playground/src/app/app.module.ts#L32-L38

**Comment by C-Dub1978** [comment]
Since i have Angular 13, i dont have ENVIRONMENT_INJECTOR and am trying to provide an alternate token


function provdieAkitaDevtools(options: Partial<DevtoolsOptions> = {}) {
  const  DEVTOOLS_INITIALIZER: InjectionToken<string> = new InjectionToken<string>('Akita Devtools');
  return {
    provide: DEVTOOLS_INITIALIZER,
    multi: true,
    useFactory() {
      return () => {
        return environment.production ? null : akitaDevtools(inject(NgZone), options)
      }
    }
  }
}


Should this work or am I bypassing functionality?
